pkg/sdkserver/localsdk_test.go

```diff
@@ -380,9 +393,10 @@ func TestLocalSDKServerPlayerConnectAndDisconnect(t *testing.T) {
 	}
 
 	for k, v := range fixtures {
+		// pin variables here, see https://github.com/kyoh86/scopelint for the details
+		k := k
+		v := v
 		t.Run(k, func(t *testing.T) {
-			t.Parallel()
```

Comment:
This line leads to unpredictable behaviour. I assume we can add this in a separate PR.